web-ui/src/backup_account/backup_email/backup_email.js

```diff
@@ -54,7 +54,8 @@ export class BackupEmail extends React.Component {
         'Content-Type': 'application/json'
       },
       body: JSON.stringify({
-        csrftoken: [browser.getCookie('XSRF-TOKEN')]
+        csrftoken: [browser.getCookie('XSRF-TOKEN')],
+        email: this.state.email
```

Comment:
Ops, it should not be here yet. 
We'll remove it for now and it will be back on the next pull request, including tests 😁